projects/components/src/data-exporter/data-exporter.component.scss

```diff
@@ -1,11 +1,71 @@
+@import '../../../../node_modules/@clr/ui/src/color/variables.color';
+@import '../../../../node_modules/@clr/ui/src/utils/variables.global';
+@import '../../../../node_modules/@clr/ui/src/typography/variables.typography';
+@import '../../../../node_modules/@clr/ui/src/layout/variables.card';
+@import '../../../../node_modules/@clr/ui/src/emphasis/alert/variables.alert';
+
 div.progress {
     visibility: hidden;
 
     &.pending {
         visibility: visible;
     }
 }
+
+.flex-row {
+    display: flex;
+    flex-direction: row;
+}
+
+.fill-space {
+    flex: 1;
+}
+
+.column-dropdown-menu {
+    height: 200px;
+    overflow: scroll;
+}
+
+.dropdown-button {
+    vertical-align: middle;
+}
+
+.progress-divider {
+    height: 20px;
+}
+
+.column-container {
+    overflow: auto;
+    height: 50px;
+    display: block;
+}
+
 button.select-all {
     margin: 0;
     padding: 0;
 }
+
+clr-tooltip-content {
+    width: 200px !important;
+}
+
+.column-container {
```

Comment:
Why do we need to fuss with the CSS? Can't we just use Clarity styles like 

```<span class="badge badge-info">2</span>```?

Reply:
This is custom style code copied from VCD-UI that makes the bubbles + X look like they look in the VMs screen. I don't know if there are any Clarity classes that apply this specific styling.